src/send/README.md

```diff
@@ -0,0 +1,27 @@
+## caliper-event-analysis: Sending the data to the OpenLRS server
+
+This script is mainly written to send caliper events that stored in the Application Database to the OpenLRS server. We are 
+manually extracting the caliper events from the App DB and the script is reading the file containing events one at a time and sends it.
+Sample caliper data file is provided in the repo and this file is Delimited by BREAK. The reason for this is some time the caliper data in the APP DB is stored as pretty-print-json and that 
+leads to adding a new line character when you do a extract. so care must be taken to get raw JSON format.The Caliper extract of each caliper event consist of Envelope info apart from the data. 
+
+### Requirements 
+   1. works with Python 2.7.10 version
```

Comment:
I plan on implementing logging instead of print. so I will put all these ideas in mind when doing this